plugin/pkg/scheduler/algorithm/priorities/interpod_affinity.go

```diff
@@ -97,6 +97,7 @@ func (p *podAffinityPriorityMap) processTerm(term *api.PodAffinityTerm, podDefin
 				if p.failureDomains.NodesHaveSameTopologyKey(node, fixedNode, term.TopologyKey) {
 					p.counts[node.Name] += weight
 				}
+				// TODO i think we need to  add processing logic for pod labels here?
```

Comment:
I don't understand this TODO. It's obviously possible that there is something wrong here, but pod labels are being processed in "PodMatchesTermsNamespaceAndSelector" function ~10 lines above.